mhctools/base_commandline_predictor.py

```diff
@@ -303,7 +310,11 @@ def predict_peptides(self, peptides):
                     allele=allele,
                     peptide_mode=True,
                     temp_dirname=temp_dirname)
-        return self._run_commands_and_collect_predictions(
+        results = self._run_commands_and_collect_predictions(
             commands=commands,
             input_filenames=input_filenames,
             temp_dir_list=dirs)
+        self._check_result_count(
```

Comment:
Usually all the peptides are missing, not just the bad ones. Seems like we should separately check for "bad" peptides (like 7mers and X). Maybe predictors can have min lengths and alphabets?